src/devtools/store.js

```diff
@@ -768,8 +795,17 @@ export default class Store extends EventEmitter {
             );
           }
 
-          element = ((this._idToElement.get(id): any): Element);
+          const element = ((this._idToElement.get(id): any): Element);
+          const prevChildren = element.children;
           element.children = Array.from(children);
+          if (element.children.length !== prevChildren.length) {
```

Comment:
Sorry. I hardened this too (to verify that no children were added or removed) and it conflicted.